app/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build:dev": "cross-env NODE_ENV=development webpack",
+    "build:prod": "cross-env NODE_ENV=production webpack",
+    "serve": "cross-env NODE_ENV=development webpack serve",
+    "test": "jest"
+  },
+  "browserslist": "> 0.25%, not dead",
+  "author": "Aliaksandra Daniliuk",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.12.16",
+    "@babel/preset-env": "^7.12.16",
+    "@babel/preset-react": "^7.12.13",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^7.0.0",
+    "cross-env": "^7.0.3",
+    "css-loader": "^5.0.2",
+    "css-minimizer-webpack-plugin": "^1.2.0",
+    "eslint": "^7.20.0",
+    "eslint-loader": "^4.0.2",
+    "file-loader": "^6.2.0",
+    "html-loader": "^1.3.2",
+    "html-webpack-plugin": "^5.1.0",
+    "jest-webpack": "^0.5.1",
+    "less": "^4.1.1",
+    "less-loader": "^8.0.0",
+    "mini-css-extract-plugin": "^1.3.6",
+    "node-sass": "^5.0.0",
+    "sass-loader": "^11.0.1",
+    "style-loader": "^2.0.0",
+    "webpack": "^5.21.2",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^3.11.2",
+    "webpack-merge": "^5.7.3"
+  },
+  "dependencies": {
+    "@babel/polyfill": "^7.12.1",
```

Comment:
@babel/polyfill is deprecated and it's never used in your app + as far as I remember it's devDependency

Reply:
Ohhh... I think it's my bad. I advised the configuration video:
![image](https://user-images.githubusercontent.com/22503765/108234414-f672ec00-716e-11eb-92bd-bdf7e64056cf.png)


@babel/polyfill is .js file, so I think it must be included as a dependency. 
And documentation says so:
![image](https://user-images.githubusercontent.com/22503765/108234179-bf9cd600-716e-11eb-8255-133a7bc69c21.png)

Reply:
Done, uninstall deprecated @babel/polyfill